### PR TITLE
Enhacements.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️
| Breaks BC?    | ❌

- [x] update `BootstrapAsset::class`.
- [x] add `BootstrapIconsAsset::class`, `BootstrapCdnAsset::class`.
- [x] fixed static analysis antotation htmlOptions in Html::addCssClass().
- [x] add finals all class.

### CODE REVIEW

File: src/Assets/BootstrapAsset.php [hunk 1]
```diff
@@ -29,10 +30,19 @@ class BootstrapAsset extends AssetBundle
         'js/bootstrap.bundle.js',
     ];
 
-    public array $publishOptions = [
-        'only' => [
-            'css/bootstrap.css',
-            'js/bootstrap.bundle.js',
-        ],
-    ];
+    public array $publishOptions = [];
```
Comment (samdark) [inline src/Assets/BootstrapAsset.php:33]: No need to define it. It's already in `AssetBundle` class.

File: src/Assets/BootstrapIconsAsset.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    public ?string $baseUrl = '@assetsUrl';
+
+    public ?string $sourcePath = '@npm/bootstrap-icons/font';
+
+    public array $css = [
+        'bootstrap-icons.css',
+    ];
+
+    public array $publishOptions = [];
```
Comment (samdark) [inline src/Assets/BootstrapIconsAsset.php:29]: No need to re-define it.

**Review by samdark — CHANGES_REQUESTED** [review]

File: src/Assets/BootstrapAsset.php (hunk 1 above)
Comment (terabytesoftw) [reply src/Assets/BootstrapAsset.php]: done.

File: src/Assets/BootstrapIconsAsset.php (hunk 2 above)
Comment (terabytesoftw) [reply src/Assets/BootstrapIconsAsset.php]: done.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Comment by Mister-42** [comment]
> add BootstrapIconsAsset::class

Why does this not have its own package?

**Comment by terabytesoftw** [comment]
> > add BootstrapIconsAsset::class
> 
> Why does this not have its own package?

it's just an asset.

**Pull request merged**